thrift_src/standard.thrift

```diff
@@ -231,6 +231,18 @@ exception InvalidCrcOperation {
  1:CrcErrorCode code
 }
 
+typedef list<i8> BmRssKey
+
+enum RssErrorCode {
```

Comment:
Here we can't do this renaming because it would result in a name collision with `ToeplitzRssErrorCode` defined in `calculations.h`. [For `custom_crc`, `CustomCrcErrorCode` and `CrcErrorCode` are used to avoid this collision in `bm_set_crc32_custom_parameters` in `Standard_servers.cpp`](https://github.com/p4lang/behavioral-model/blob/a2e920781f72631159cc4a1bd591787c8b44c8ac/src/bm_runtime/Standard_server.cpp#L1160-L1174). I followed that example to use the shorthand `RssErrorCode` here. But if you have a better name in mind, please let me know.

Reply:
let's use the `ToeplitzRssHash` prefix in one place and the `ToeplitzRss` prefix in the other